cibyl/cli/query.py

```diff
@@ -35,6 +35,61 @@ class QueryType(IntEnum):
     """Retrieve data concerning builds and above."""
 
 
+class QuerySelector:
+    """Deduce the type of query performed from the cli argument considering
+    both core argument and plugin provided ones."""
+    query_selector_functions = []
+
+    def get_query_type_core(self, **kwargs):
```

Comment:
You can make this one private, what need we have for it.

Reply:
Good point